examples/text-generation/run_generation.py

```diff
@@ -260,6 +260,11 @@ def setup_parser(parser):
         action="store_true",
         help="Whether to enable device map auto. In case no space left on cpu, weights will be offloaded to disk.",
     )
+    parser.add_argument(
+        "--trust_remote_code",
+        action="store_true",
+        help="Whether or not to allow for custom models defined on the Hub in their own modeling files.",
```

Comment:
Maybe could be reworded a bit.

Like: `Allow custom models defined on the Hub in their own modeling files`

Also the files could be local path, so doesn't have to be from the hub. So maybe: `Allow models with their own modeling files outside transformers repo`

Reply:
Sorry for the latency, the help message here come from huggingface doc: https://huggingface.co/docs/transformers/model_doc/auto#transformers.AutoConfig.from_pretrained.trust_remote_code